content/kubernetes/active-active/create-reaadb.md

```diff
@@ -120,10 +120,13 @@ For a list of example values used throughout this article, see the [Example valu
          databaseSecretName: <my-secret>
          memorySize: 200MB
          shardCount: 3
+         tlsMode: enabled 
```

Comment:
please view the slack thread linked to the issue.
we use the default tlsMode that CRDB is using and it is working on any env.